profile/benchmark_helper.rb

```diff
@@ -0,0 +1,194 @@
+require 'mongo'
+require 'json'
+require 'FileUtils'
```

Comment:
this is making our travis ci tests fail: https://travis-ci.org/mongodb/mongo-ruby-driver/builds/93085864